### ISSUE TITLE
model tests

### ISSUE THREAD

**Issue by jaentrouble** [jaentrouble/Adipose_tf_train3#1]

**Comment by jaentrouble** [comment]
### hr_5_3_0_d4
![image](https://user-images.githubusercontent.com/45917844/92318695-40499900-f04b-11ea-91dd-5f39d314eb12.png)

Loss overshooting.

**Comment by jaentrouble** [comment]
### hr_5_3_0_d128
- Added some batch normalization at the end

![image](https://user-images.githubusercontent.com/45917844/92318711-5bb4a400-f04b-11ea-82ad-a59625abe8bc.png)

Loss overshooting again.
Need to normalize outputs (box values to be 0.0~1.0)

**Comment by jaentrouble** [comment]
### hr_5_3_0_d128_normoutpos
- Normalized output and pos to 0.0~1.0
![image](https://user-images.githubusercontent.com/45917844/92323888-376ebc80-f077-11ea-8d7b-a41ed91a40a1.png)

Loss converging smoothly.
~Need to visualize~

- Blue : Target
- Red : predict
![hr_5_3_0_d128_normoutpos](https://user-images.githubusercontent.com/45917844/92326095-8114d280-f08a-11ea-813b-2ddda52c895b.png)

**Comment by jaentrouble** [comment]
## Change pos to normalized pos
The image size has to be changed anyway, so training with normalized position (between 0 to 1) is better.

**Comment by jaentrouble** [comment]
### hr_5_3_0_d256_e128

- Blue : training
- Pink : validation
*NOTE: the validation image uses the whole image, which makes the box smaller. Therefore, the loss value is smaller than training set*
![image](https://user-images.githubusercontent.com/45917844/92425704-ff749000-f1c2-11ea-97dd-7dc662eaf919.png)


![image](https://user-images.githubusercontent.com/45917844/92425572-99880880-f1c2-11ea-8d9e-be95b6d8a87b.png)

- Still not so good.

**Comment by jaentrouble** [comment]
### hr_5_3_16_d256_e128
**Changed image size to (480,320)**

- Twiced the number of every filters.

- Green : training
- Gray : validation
*NOTE: the validation image uses the whole image, which makes the box smaller. Therefore, the loss value is smaller than training set*
![image](https://user-images.githubusercontent.com/45917844/92425856-61cd9080-f1c3-11ea-920f-e2cc6fd0c6d8.png)

![hr_5_3_16_d256_e128](https://user-images.githubusercontent.com/45917844/92426268-79594900-f1c4-11ea-9cb6-deb54fb2ddaa.png)

- Even worse performance

**Comment by jaentrouble** [comment]
### Abandon
